roles/core/cluster/defaults/main.yml

```diff
@@ -25,6 +25,9 @@
 ## Spectrum Scale daemon nodename (defaults to node's hostname)
 scale_daemon_nodename: "{{ ansible_hostname }}"
 
+## Spectrum Scale admin nodename (defaults to node's hostname)
+scale_admin_nodename: "{{ ansible_hostname }}"
```

Comment:
You might want to set `scale_admin_nodename` to `scale_daemon_nodename` by default. If users change daemon network then admin network will follow these changes automatically...
```
scale_admin_nodename: "{{ scale_daemon_nodename }}"
```